isobus/include/isobus/isobus/isobus_diagnostic_protocol.hpp

```diff
@@ -90,7 +90,7 @@ namespace isobus
 			DataValidAboveNormalModeratelySevere = 16, ///< Condition is above what would be considered normal as determined by the predefined moderately severe level limits for that particular measure of the condition
 			DataValidBelowNormalLeastSevere = 17, ///< Condition is below what would be considered normal as determined by the predefined least severe level limits for that particular measure of the condition
 			DataValidBelowNormalModeratelySevere = 18, ///< Condition is below what would be considered normal as determined by the predefined moderately severe level limits for that particular measure of the condition
-			ReceivedNetworkDataInError = 19, ///< Any failure that is detected when the data received through the network are found replaced by the �error indicator� value 0xFE
+			ReceivedNetworkDataInError = 19, ///< Any failure that is detected when the data received through the network are found replaced by the �error indicator� value 0xFE
```

Comment:
Not sure what happened here, something got corrupted saving this on Linux. Looks like the original quotes are windows quotes or something? I'll need to fix this to use the `"` character

Reply:
I can comb through it later and fix these, it seems like there's more than I expected